datus-sqlalchemy/datus_sqlalchemy/connector.py

```
@@ -0,0 +1,644 @@
+        try:
+            samples = []
+            if not tables:
+                tables = []
+                if table_type in ("table", "full"):
+                    tables.extend(self.get_tables(catalog_name, database_name, schema_name))
+                if table_type in ("view", "full"):
+                    tables.extend(self.get_views(catalog_name, database_name, schema_name))
+                if table_type in ("mv", "full"):
+                    try:
+                        tables.extend(self.get_materialized_views(catalog_name, database_name, schema_name))
+                    except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)